app/1.0/start/toolbox/add-elements.md

```diff
@@ -1,84 +1,71 @@
 ---
-title: Step 3. Add some elements
+title: Step 3. Add an element
 subtitle: "Build an app with App Toolbox"
 ---
 
 <!-- toc -->
 
-Now that you've added a new view to your application, you can start building
-out the details of that view.
+One of the appeals of the web component approach to web development 
+is the ecosystem. Rather than building everything in your app from scratch,
+you can find and use third-party elements that suit your needs. Two examples
```

Comment:
"third-party" isn't correct here (that suggests a component built by someone who isn't us and isn't the developer we're talking to). Could use 'ready-made' or 'off-the-shelf'.